cwms_radar_api/src/test/java/cwms/radar/api/LocationGroupControllerTestIT.java

```diff
@@ -49,7 +49,7 @@ class LocationGroupControllerTestIT extends DataApiTestIT
 
 	@Test
 	void test_create_read_delete() throws Exception {
-		String officeId = RadarApiSetupCallback.getDatabaseLink().getOfficeId();
```

Comment:
When the automated tests run, they are authorized for all offices and this one is set to HQ?

Reply:
No the account has specific offices in this case just SPK, but the connecting user has the HQ priv. That's basically how the "central writable" system will need to work.

Reply:
ok, so I should make sure to always use SPK

Reply:
Yeah. Or at the least it should be specific. I'm thinking of making the "TestAccount" users hold a set of offices so it can also be looped over, but I'm not quite sure how that'll work.... yet.